app/models/user.rb

```diff
@@ -9,4 +11,8 @@ class User < ActiveRecord::Base
     :rememberable,
     :trackable,
     :validatable
+
+  def full_name
+    "#{first_name} #{last_name}"
+  end
```

Comment:
Great practice to use helper methods like this one. Just need to add a super quick model unit test to complete this PR!